### DISCUSSION TITLE
Inserting pictures inline after drop causes problem when inline is not set to true

### DISCUSSION THREAD

**Discussion by michaelklopf** [ueberdosis/tiptap#1635] [General Questions]
Hi,

we've a rather complex problem. We fetch the drop on the Texteditor, show a modal dialog and let the user choose if he wants to insert pictures either inline or as attachment parallel to the editor.

The problem is, when using block and not inline, that dropping multiple image files causes a problem. Only one image is inserted when calling

```js
editor.value
  .chain()
  .focus()
  .setImage({ src: res.target.result })
  .run();
```

multiple times.

Maybe the problem is the same as here in the images example https://www.tiptap.dev/examples/images/

Add a couple of new lines and try to insert pictures with dragging and dropping them where you can see the cursor in the screenshot. This does nothing, only when dropping the pictures exactly above the other pictures.
<img width="821" alt="image" src="https://user-images.githubusercontent.com/2892138/126806979-0dc241da-6787-4cde-bf19-eb16b45fa38b.png">

We guess that it is a bug in the editor. Trying to do that in Brave/Chrome fails completely, dropped pictures are opened in a new tab.

**Comment by philippkuehn** [comment]
See #2142
